src/components/TeacherComponents/SchoolDetailsInputBox/SchoolDetailsInputBox.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { OakFlex, OakSpan } from "@oaknational/oak-components";
+
+import { OakColorName } from "@/styles/theme";
+import {
+  RotatedInputLabel,
+  StyledInput,
+} from "@/components/SharedComponents/Input/Input";
+import BoxBorders from "@/components/SharedComponents/SpriteSheet/BrushSvgs/BoxBorders";
+
+type SchoolDetailsInputBoxProps = {
+  placeholder: string;
+  background: OakColorName;
+  labelText: string;
+  hasError?: boolean;
+  onBlur: (e: React.FocusEvent<HTMLInputElement>) => void;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;
+};
+
+const SchoolDetailsInputBox: React.FC<SchoolDetailsInputBoxProps> = ({
+  placeholder,
+  background,
+  labelText,
+  onBlur,
+  onChange,
+  hasError,
+}) => {
+  return (
+    <OakFlex $width={"100%"} $position={"relative"} $display={"inline-block"}>
+      <BoxBorders gapPosition="rightTop" />
+      <OakFlex $position={"absolute"}>
```

Comment:
is this not the same as the `Input` component?

Reply:
Changed to Input component ✅